spaceship/lib/spaceship/portal/provisioning_profile.rb

```diff
@@ -265,7 +262,11 @@ def all(mac: false, xcode: false)
           end
 
           # filter out the profiles managed by xcode
-          profiles.delete_if(&:managed_by_xcode?) unless xcode
+          if xcode
+            warn('Apple API no longer returns XCode managed Provisioning Profiles')
```

Comment:
This is a breaking change that we don't have a way to get around. The API no longer returns profiles that are managed by xcode so this param does nothing now.